micro_sam/training/util.py

```diff
@@ -14,12 +14,13 @@ def get_trainable_sam_model(
     device: Optional[Union[str, torch.device]] = None,
     checkpoint_path: Optional[Union[str, os.PathLike]] = None,
     freeze: Optional[List[str]] = None,
-    checkpoint_path: Optional[Union[str, os.PathLike]] = None,
```

Comment:
Are you removing this line on purpose, @constantinpape ?

Reply:
Yes. That's on purpose, It somehow has been duplicated (see line 15 there is also the argument `checkpoint_path`). So this is just a small bugfix. (I am not sure how this got introduced, maybe in a recent merge.)